pgjdbc/src/main/java/org/postgresql/core/v3/QueryExecutorImpl.java

```diff
@@ -2370,7 +2370,7 @@ protected void processResults(ResultHandler handler, int flags) throws IOExcepti
           // In case of CopyOutResponse, we cannot abort data transfer,
           // so just throw an error and ignore CopyData messages
           handler.handleError(
-              new PSQLException(GT.tr("The driver currently does not support COPY operations."),
+              new PSQLException(GT.tr(COPY_ERROR_MESSAGE),
```

Comment:
Ok. Now I have a technical justification for reverting the change: it breaks translation, as the string no longer considered translatable.